generic_tracers/generic_COBALT.F90

```diff
@@ -6563,8 +6672,6 @@ subroutine user_allocate_arrays
     allocate(cobalt%jnamx(isd:ied, jsd:jed, 1:nk)) ; cobalt%jnamx=0.0
     allocate(cobalt%omega_arag(isd:ied, jsd:jed, 1:nk))   ; cobalt%omega_arag=0.0
     allocate(cobalt%omega_calc(isd:ied, jsd:jed, 1:nk))   ; cobalt%omega_calc=0.0
-    allocate(cobalt%omegaa(isd:ied, jsd:jed, 1:nk))       ; cobalt%omegaa=0.0
```

Comment:
We should also remove `omegaa` and `omegac` from `cobalt_types.F90` if they are not used:

https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/blob/b5203ce9cb4f52c9674696bbd0e5d672676a087e/generic_tracers/cobalt_types.F90#L718-L719

Reply:
Agreed.  I will add this along with the various fixes above to another commit.